launch/velocity_smoother-launch.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+# All rights reserved.
+#
+# Software License Agreement (BSD License 2.0)
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# * Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above
+#   copyright notice, this list of conditions and the following
+#   disclaimer in the documentation and/or other materials provided
+#   with the distribution.
+# * Neither the name of {copyright_holder} nor the names of its
+#   contributors may be used to endorse or promote products derived
+#   from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+"""Launch the velocity smoother node with default configuration."""
+
+import os
+
+import ament_index_python.packages
+import launch
+import launch_ros.actions
+
+
+def generate_launch_description():
+    share_dir = ament_index_python.packages.get_package_share_directory('velocity_smoother')
+    params_file = os.path.join(share_dir, 'config', 'velocity_smoother_params.yaml')
+    velocity_smoother_node = launch_ros.actions.Node(package='velocity_smoother',
+                                                      node_executable='velocity_smoother_node',
+                                                      output='both',
+                                                      parameters=[params_file])
+
+    return launch.LaunchDescription([velocity_smoother_node,
+
+                                     launch.actions.RegisterEventHandler(
+                                         event_handler=launch.event_handlers.OnProcessExit(
+                                             target_action=velocity_smoother_node,
+                                             on_exit=[launch.actions.EmitEvent(
+                                                 event=launch.events.Shutdown())],
```

Comment:
When is this needed? I haven't been adding it on any of my nodes to date.

Reply:
I actually don't know.  It was something I picked up from the launch examples and have been adding to my launch files ever since.

Reply:
Try dropping it. If we're not using it, we're just creating noise for users.

Reply:
Dropped in ec95262